### PR TITLE
Mimir 697 automatic data fetch on create

### PR DESCRIPTION
**Pull request by itemodahl**

### CODE REVIEW

**Comment by prabu-ssb** [comment]
Such a lovely PR!

File: src/main/resources/lib/listeners.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Content, QueryResponse } from 'enonic-types/lib/content'
+import { HighchartConfig } from '../site/macros/highchart/highchart-config'
+import { KeyFigure } from '../site/content-types/keyFigure/keyFigure'
+import { EnonicEvent } from 'enonic-types/lib/event'
+import {DatasetRepoNode} from "./repo/dataset";
```
Comment (prabu-ssb) [inline src/main/resources/lib/listeners.ts:5]: lint?

File: src/main/resources/lib/listeners.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        query: `_id = '${event.data.nodes[0].id}' AND 
+          ( data.dataSource._selected = '${DataSource.STATBANK_API}' OR data.dataSource._selected = '${DataSource.TBPROCESSOR}' )`,
+        contentTypes: [`${app.name}:highchart`, `${app.name}:keyFigure`],
+        filters: {
+          exists: {
+            field: `data.dataSource.*.urlOrId`
+          }
+        }
+      })
+      contentWithDataSource.hits.forEach((content: Content<HighchartConfig | KeyFigure>) =>  refreshDataset(content))
```
Comment (prabu-ssb) [inline src/main/resources/lib/listeners.ts:41]: not a candidate for async fetch ?

Comment (itemodahl) [reply src/main/resources/lib/listeners.ts]: Not really :) It will always be one (probably).

File: src/main/resources/lib/listeners.ts (hunk 1 above)
Comment (itemodahl) [reply src/main/resources/lib/listeners.ts]: Linted :)

**Pull request merged**